docs/devel/development.md

```diff
@@ -50,6 +50,14 @@ Official releases are built using Docker containers. To build Kubernetes using
 Docker please follow [these
 instructions](http://releases.k8s.io/HEAD/build/README.md).
 
+### Local OS/shell environment
+
+Many of the Kubernetes development helper scripts rely on a fairly up-to-date GNU tools
```

Comment:
Then maybe reword "It requires <xxx> version. And note that Mac OS X may not ship with this by default."